pipeline/README.md

```diff
@@ -87,3 +93,11 @@ The EC2 instance running the mhs build (to run the integration tests against) mu
 established using the Fabric script in `scripts`. The open test certs and party key files should be copied
 to the EC2 instance in a `home/ec2-user/certs` directory (this directory is a shared volume with the running 
 mhs docker container as specified in the docker container). 
+
+
+There must also be log groups in cloud watch under the following names:
```

Comment:
```suggestion
There must also be log groups in cloudwatch under the following names:
```

And do these need to be created manually, I don't recall doing that (unless it was the AWS wizard doing that for me).